src/components/Auth/ErrorNotification/ErrorNotication.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import cN from 'classnames';
+import { Error } from '../../../types/Errors';
+
+type Props = {
+  onErrorChange: (isError: Error | null) => void,
+  errors: Error,
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  errors,
+  onErrorChange,
+}) => {
+  return (
```

Comment:
```suggestion
}) => (
```
you may use shorthand here